builtin/settingtypes.txt

```diff
@@ -828,25 +837,38 @@ tooltip_append_itemname (Append item name) bool false
 #    Whether FreeType fonts are used, requires FreeType support to be compiled in.
 freetype (FreeType fonts) bool true
 
-#    Path to TrueTypeFont or bitmap.
+#    Path to the default font.
+#    Must be a TrueTypeFont or bitmap font.
 font_path (Font path) filepath fonts/liberationsans.ttf
 
+#    Font size of the default font in point (pt).
 font_size (Font size) int 16
 
-#    Font shadow offset, if 0 then shadow will not be drawn.
+#    Shadow offset (in pixels) of the default font. If 0, then shadow will not be drawn.
```

Comment:
I didn't even know one could adjust the shadow offset… The more I know :slightly_smiling_face: